youtube_dl/extractor/svt.py

```diff
@@ -141,21 +146,28 @@ class SVTPlayIE(SVTPlayBaseIE):
                     )
                     '''
     _TESTS = [{
-        'url': 'http://www.svtplay.se/video/5996901/flygplan-till-haile-selassie/flygplan-till-haile-selassie-2',
-        'md5': '2b6704fe4a28801e1a098bbf3c5ac611',
+        'url': 'https://www.svtplay.se/video/13906852/en-liten-film-om/en-liten-film-torsk-pa-tallinn-en-liten-film-om-ensamhet',
+        'md5': '2382036fd6f8c994856c323fe51c426e',
         'info_dict': {
-            'id': '5996901',
+            'id': 'jvR32zj',
             'ext': 'mp4',
-            'title': 'Flygplan till Haile Selassie',
-            'duration': 3527,
-            'thumbnail': r're:^https?://.*[\.-]jpg$',
+            'title': 'Torsk på Tallinn - en liten film om ensamhet',
+            'timestamp': 1366063200,
+            'upload_date': '20130415',
+            'duration': 3510,
+            'thumbnail': r're:^https?://(?:.*[\.-]jpg|www.svtstatic.se/image/.*)$',
             'age_limit': 0,
             'subtitles': {
                 'sv': [{
+                    'ext': 'vtt',
+                }, {
                     'ext': 'wsrt',
                 }]
             },
         },
+        'params': {
+            'format': 'bestvideo',
+        },
```

Comment:
Without this i get `DownloadError: ERROR: requested format not available` when running test. Not sure why. I also noticed that with another test video https://www.svtplay.se/video/26595950/we-got-this/we-got-this-sasong-1-katten-rio i got an download error about `AssertionError: Expected test_SVTPlay_jNwpvBJ.mp4 to be at least 9.77KiB, but it's only 865.00B` which seems to be because the first init segment in the dash stream is 800 byte, tried to change `file_minsize` but the lowest seems to be 1000 because of https://github.com/ytdl-org/youtube-dl/blob/master/test/test_download.py#L219